src/sentry/api/serializers/models/project.py

```diff
@@ -184,7 +184,7 @@ def format_options(attrs: defaultdict(dict)):
         f"filters:{FilterTypes.ERROR_MESSAGES}": "\n".join(
             attrs["options"].get(f"sentry:{FilterTypes.ERROR_MESSAGES}", [])
         ),
-        "feedback:branding": attrs.get("feedback:branding", "1") == "1",
+        "feedback:branding": attrs["options"].get("feedback:branding", "1") == "1",
```

Comment:
I can't comment on it, but is `"sentry:reprocessing_active": bool(attrs.get("sentry:reprocessing_active", False)),` also broken?

Reply:
Maybe we should do
```
options = attrs["options"]
....
    options.get(...)
```

Might be less error prone

Reply:
i think you're right (on both counts)